### ISSUE TITLE
Adding UI for MEVN-cli

### ISSUE THREAD

**Issue by jamesgeorge007** [madlabsinc/mevn-cli#67]
With this we plan to create a ui for MEVN-cli tool which will help developers to add packages and integrations, basically execute different commands used in cli via a ui which would be running locally.

**Comment by IsAmrish** [comment]
@jamesgeorge007 can you give some ref for it?

**Comment by jamesgeorge007** [comment]
@IsAmrish `vue-cli` has got a `ui` command which allows the user to set up their project from a web-based UI. We're gonna follow a similar approach.

**Comment by nasr18** [comment]
its not that much simple. need complete front-end app. right?

**Comment by jamesgeorge007** [comment]
@nasr18 yeah, it is a bit complex and requires to have collaborative effort.

**Comment by peoray** [comment]
What's the status on this? I'll like to contribute.

**Comment by jamesgeorge007** [comment]
Sure, feel free.

**Comment by pradsdev** [comment]
I would like to contribute, Does anybody have any plans or ideas to start with?

**Comment by slzakaria** [comment]
Hello ! i would love to help on this task, what's the current situation of the ui part ?
